aiohttp/client_reqrep.py

```diff
@@ -147,7 +146,7 @@ def check(self, transport: asyncio.Transport) -> None:
 
 
 if ssl is not None:
-    SSL_ALLOWED_TYPES = (ssl.SSLContext, bool, Fingerprint, type(None))
+    SSL_ALLOWED_TYPES = (ssl.SSLContext, bool, Fingerprint)
 else:  # pragma: no cover
     SSL_ALLOWED_TYPES = type(None)
```

Comment:
Also, this presumably needs to support bool now.
```suggestion
    SSL_ALLOWED_TYPES = (bool, type(None))
```